_posts/nataliele/2016-01-14-nataliele-first-post.md

```diff
@@ -1,6 +1,7 @@
 ---
 layout: post
 author: nataliele
+title: Natalie's first post
```

Comment:
@nataliele this `'` will make Jekyll freak out.  Enclosing the title in `"` will force it to realize it's text, not some crazy code.